include/triton/Dialect/TritonGPU/IR/LinearLayoutConversions.h

```diff
@@ -113,12 +113,22 @@ LinearLayout chooseShemLayoutForRegToRegConversion(
 //   row0  reg[0-1]   reg[4-5]
 //   row8  reg[2-3]   reg[6-7]
 //
+// When swizzleByteSize is non-zero, the layout is constructed differently.
+// In the swizzled layout, the leading dimension (i.e., column dimension) is
+// strided by swizzleByteSize.  For example, in a matrix of size 128x128 with a
+// data type of f16, stored in shared memory using 128B-swizzle mode, the offset
+// of the element at index (1, 0) will be 72 due to the stride.  Without
```

Comment:
If you have a 128x128 matrix with no swizzling, I would have thought that the element at index (1,0) would be at offset 128.  How do we get 64 and 72?

(I also don't see how this thing has to do with swizzling.  As described is it just the stride of dimension 1, measured in bytes?)

Reply:
> If you have a 128x128 matrix with no swizzling, I would have thought that the element at index (1,0) would be at offset 128. How do we get 64 and 72?

My bad... I'll just remove the offset without swizzling.

Reply:
72 should be corrected with 64 + 2 bytes/element * 8 swizzled coordinate = 80 if we use the number of bytes as the unit of offset.